tools/old/stackcount.py

```diff
@@ -145,10 +145,7 @@ def print_frame(addr):
     print("  ", end="")
     if verbose:
         print("%-16x " % addr, end="")
-    if offset:
-        print("%s" % b.ksymaddr(addr))
-    else:
-        print("%s" % b.ksym(addr))
+    print(b.ksym(addr, show_address=offset))
```

Comment:
`=True`. It doesn't look like `offset` does anything special so seeing this instead of `True` may confuse.